api/src/plugins/cors.ts

```diff
@@ -24,7 +24,7 @@ const cors: FastifyPluginCallback = (fastify, _options, done) => {
     void reply
       .header(
         'Access-Control-Allow-Headers',
-        'Origin, X-Requested-With, Content-Type, Accept, Csrf-Token'
+        'Origin, X-Requested-With, Content-Type, Accept, Csrf-Token, Coderoad-User-Token, Exam-Environment-Authorization-Token'
```

Comment:
Any idea why Coderoad-User-Token was working without this? Did it just not get ported over to the new api?

Reply:
As far as I saw, this works on the same domain with/without the access control. Maybe because cors does not apply to the same domain 🤷‍♂️ 

So, if we served swagger or the tests on a different domain to the server, then they would fail.